index.html

```diff
@@ -6,10 +6,48 @@
     <link rel="stylesheet" href="style.css" />
     <link rel="stylesheet" media=" all and (min-width:481px) and (max-width:1024px)" href="tablet.css" />
```

Comment:
Pour info, il y a différents breakpoints, t'es libre de choisir ce qui t'arrange, tu peux trouver plus de détail ici
https://css-tricks.com/snippets/css/media-queries-for-standard-devices/
https://stackoverflow.com/questions/6370690/media-queries-how-to-target-desktop-tablet-and-mobile